### PR TITLE
README update

### PR DESCRIPTION
**Pull request by mgcam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+     QC checks and storage of QC metrics
+ - npg_qc::mqc - evaluation an dreporting of QC metrics
+
+### bin
+
+Perl scripts which are deployed to the production environment
+
+### scripts
+
+Supplimentary scripts, not deployed
```
Comment (nerdstrike) [inline README.md:19]: s/Supplimentary/Supplementary/

File: README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+### t
+
+Tests, test data, supplimentary scripts and modules for testing
```
Comment (nerdstrike) [inline README.md:23]: s/Supplimentary/Supplementary/

File: README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+### docs
+
+Documentation, useful code snippets
+
+## NPG QC database
+
+### Legacy Data
+
+QC metrics and evaluation outcomes for all production Illumina sequensing runs
```
Comment (nerdstrike) [inline README.md:37]: s/sequensing/sequencing/

**Review by nerdstrike — CHANGES_REQUESTED** [review]
Positive change. Can we fix the spelling while we're here too?

**Review by dkj — APPROVED** [review]
Like - but ideally with @nerdstrike 's spelling fixes.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+     QC checks and storage of QC metrics
+ - npg_qc::mqc - evaluation and reporting of QC metrics
+
+### bin
+
+Perl scripts which are deployed to the production environment
+
+### scripts
+
+Suppliementary scripts, not deployed
```
Comment (dkj) [inline README.md:19]:
```suggestion
Supplementary scripts, not deployed
```

Comment (mgcam) [reply README.md]: fixed

**Pull request merged**